benchmarks/benchmarks.py

```diff
@@ -0,0 +1,36 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+
+
+class TimeSuite:
```

Comment:
As a start, I'm imagining having a Suite for both of the stable-ish subpackages - `AtomicSuite` and `PhysicsSuite` - and then we could just copy and paste some function executions from a few sample docstrings or gallery examples.

Reply:
How are we going to handle the new carpet .asv , I added a line to ignore ir because it creates a whole venv but in the config file this lines appear
`    "env_dir": ".asv/env",
    "results_dir": ".asv/results",
    "html_dir": ".asv/html"`